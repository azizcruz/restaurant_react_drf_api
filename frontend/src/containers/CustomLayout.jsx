import React from "react";
import { Layout, PageHeader } from "antd";

const { Header, Content, Footer } = Layout;

class CustomLayout extends React.Component {
  render() {
    return (
      <Layout className="layout">
        <Header>
          <h1 style={{ color: "#fff" }} >Restaurant Menu</h1>
        </Header>
        <Content style={{ padding: "0 50px" }}>
          <div style={{ padding: "24px 0", minHeight: 84 + "vh" }}>
            {this.props.children}
          </div>
        </Content>
        <Footer style={{ textAlign: "center" }}>Abdelaziz Abdelioua</Footer>
      </Layout>
    );
  }
}

export default CustomLayout;
