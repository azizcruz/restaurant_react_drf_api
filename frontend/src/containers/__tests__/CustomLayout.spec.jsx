import React from "react";
import { create } from "react-test-renderer";
import CustomLayout from "../CustomLayout";

describe("CustomLayout component", () => {
    test("it matches the snapshot", () => {
      const component = create(<CustomLayout />);
      expect(component.toJSON()).toMatchSnapshot();
    });
  });