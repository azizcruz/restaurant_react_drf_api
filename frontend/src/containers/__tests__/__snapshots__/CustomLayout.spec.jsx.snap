// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CustomLayout component it matches the snapshot 1`] = `
<section
  className="layout ant-layout"
>
  <header
    className="ant-layout-header"
  >
    <h1
      style={
        Object {
          "color": "#fff",
        }
      }
    >
      Restaurant Menu
    </h1>
  </header>
  <main
    className="ant-layout-content"
    style={
      Object {
        "padding": "0 50px",
      }
    }
  >
    <div
      style={
        Object {
          "minHeight": "84vh",
          "padding": "24px 0",
        }
      }
    />
  </main>
  <footer
    className="ant-layout-footer"
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Abdelaziz Abdelioua
  </footer>
</section>
`;

exports[`DataTable component it matches the snapshot 1`] = `
<section
  className="layout ant-layout"
>
  <header
    className="ant-layout-header"
  >
    <h1
      style={
        Object {
          "color": "#fff",
        }
      }
    >
      Restaurant Menu
    </h1>
  </header>
  <main
    className="ant-layout-content"
    style={
      Object {
        "padding": "0 50px",
      }
    }
  >
    <div
      style={
        Object {
          "minHeight": "84vh",
          "padding": "24px 0",
        }
      }
    />
  </main>
  <footer
    className="ant-layout-footer"
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    Abdelaziz Abdelioua
  </footer>
</section>
`;
